presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaTableDescriptionSupplier.java

```diff
@@ -124,4 +126,14 @@
         }
         return ImmutableList.of();
     }
+
+    private static SchemaTableName parseTableName(String schemaTableName)
+    {
+        checkArgument(!isNullOrEmpty(schemaTableName), "schemaTableName is null or is empty");
+        String[] parts = schemaTableName.split("\\.", -1);
+        if (parts.length != 2) {
```

Comment:
Use `checkArgument` for this. We didn't use it in `SchemaTableName` because the SPI doesn't depend on Guava.